.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest a feature or improvement
+title: "[FEATURE]: "
+labels: kind/enhancement
+assignees: ''
+
+---
+
+<!-- MAKE SURE TO APPLY SUITABLE LABELS BEFORE FILING THE ISSUE, ESPECIALLY PRIORITY -->
+<!-- AND `kind/documentation` IF IT INVOLVES DOCUMENTATION IN ANY WAY e.g. README -->
+## Details
+
+**Is your feature request related to a problem? Please describe.**
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+**Describe the solution you'd like**
```

Comment:
We should not be asking about a solution but rather use-case, or expected workflow.

Reply:
@adelton Please make a suggestion on what you want this sentence to be.

Reply:
"Describe the use-case or expected workflow." :-)

Reply:
```suggestion
**Describe the use-case or expected workflow**
```